docs/integrations/destinations/exasol.md

```diff
@@ -0,0 +1,72 @@
+# Exasol
+
+Exasol is the in-memory database built for analytics.
+
+## Sync overview
+
+### Output schema
+
+Exasol tables become Airbyte Streams and Exasol columns become Airbyte Fields. Each table will contain 3 columns:
+
+* `_AIRBYTE_AB_ID`: a uuid assigned by Airbyte to each event that is processed. The column type in Exasol is `VARCHAR(64)`.
+* `_AIRBYTE_DATA`: a json blob representing with the event data. The column type in Exasol is `VARCHAR(2000000)`.
+* `_AIRBYTE_EMITTED_AT`: a timestamp representing when the event was pulled from the data source. The column type in Exasol is `TIMESTAMP`.
+
+
+### Data type mapping
+
+This section should contain a table mapping each of the connector's data types to Airbyte types. At the moment, Airbyte uses the same types used by [JSONSchema](https://json-schema.org/understanding-json-schema/reference/index.html). `string`, `date-time`, `object`, `array`, `boolean`, `integer`, and `number` are the most commonly used data types.
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+
+
+### Features
+
+This section should contain a table with the following format:
+
+| Feature | Supported? (Yes/No) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync |  |  |
+| Incremental Sync |  |  |
+| Replicate Incremental Deletes |  |  |
+| For databases, WAL/Logical replication |  |  |
+| SSL connection | Yes | TLS |
+| SSH Tunnel Support | No |  |
+
+### Performance considerations
+
+## Getting started
+
+### Requirements
+
+To use the Exasol destination, you'll need Exasol version 7.1 or above.
+
+#### Network Access
+
+Make sure your Exasol database can be accessed by Airbyte. If your database is within a VPC, you may need to allow access from the IP you're using to expose Airbyte.
+
+#### **Permissions**
+
+As Airbyte namespaces allows us to store data into different schemas, we have different scenarios and list of required permissions:
+
+| Login user | Destination user | Required permissions | Comment |
+| :--- | :--- | :--- | :--- |
+| DBA User | Any user | - |  |
+| Regular user | Same user as login | Create, drop and write table, create session |  |
```

Comment:
What is the difference between "table" and "ANY table"?

Reply:
"ANY table" grants permissions for all tables in all schemas.